plugins/MultiNull/src/Logging.cpp

```diff
@@ -0,0 +1,49 @@
+
```

Comment:
I don't know what the PluginLog object API is.

Also, the void setConfigDir(loggingInfo* li, const char* confdir) is there for later.
Currently logging is broken because I forgot to put a malloc somewhere. (I should do that later)